numpy/core/src/multiarray/shape.c

```diff
@@ -108,7 +108,7 @@ PyArray_Resize(PyArrayObject *self, PyArray_Dims *newshape, int refcheck,
             PyErr_SetString(PyExc_ValueError,
                     "cannot resize an array that "
                     "references or is referenced\n"
-                    "by another array in this way.  Use the resize function");
+                    "by another array in this way.  Use the np.resize function or refcheck=False");
```

Comment:
This is not correct - passing `refcheck = False` doesn't help if `PyArray_BASE(arr)` is set.

Either:
* remove the remark about refcheck
* Split this `if` into two ifs, and only mention refcheck in the one where `refcnt > 2` is true.